### DISCUSSION TITLE
About ElevenClock update cycles

### DISCUSSION THREAD

**Discussion by marticliment** [martinet101/ElevenClock#168] [General]
## 🌐 About update cicle!
  From this release, ElevenClock's versions will have 5 or more days between updates, and those periods are going to be longer and longer.

  This happens because on the first stages of the application (until version 2.1.1), ElevenClock was full of important bugs that had to be fixed. Because those important bugs were solved and current bugs impact less people and thay have less severity, updates will be more distant, because there's no general urgency that affects much people. 

  If it appears a bug/error that impacts lots of people or has a high severity, i'll fix it and release a fix ASAP (like what happened with 2.1 and 2.1.1). 

 This change in the update cycle is to prevent updates number to be insane (like it happened before).

**Comment by marticliment** [comment]
For those days I'll switch to 2 updates per month (1 update every two weeks), but I have plans to return to the update-per-week cycle

**Comment by SoCalDude** [comment]
Does the app automatically check for updates?

**Reply by marticliment** [reply]
Yes, it does.

However, due to some issues with the old domain, ElevenClock versions 1.x.x, 2.x.x and 3.x.x are not capable of updating themselves, so a reinstall will be required (ElevenClock can be easily reinstalled from the [Microsoft Store](https://apps.microsoft.com/store/detail/elevenclock/XP89HZ8SVWTT0M)). ElevenClock versions 4.x.x should update themselves automatically.

**Reply by SoCalDude** [reply]
Thanks, Marti. I just installed it from the latest on Github (v4.1.0), so it looks like it will self-update. I have another question, but I will start another topic. Thank you again.
